packages/runtime/datastore/src/dataStoreRuntime.ts

```diff
@@ -708,13 +708,7 @@ IFluidDataStoreChannel, IFluidDataStoreRuntime, IFluidHandleContext {
                 // (i.e. it has a base mapping) - then we go ahead and summarize
                 return isAttached;
             }).map(async ([contextId, context]) => {
-                // If BlobAggregationStorage is engaged, we have to write full summary for data stores
-                // BlobAggregationStorage relies on this behavior, as it aggregates blobs across DDSs.
-                // Not generating full summary will mean data loss, as we will overwrite aggregate blob in new summary,
-                // and any virtual blobs that stayed (for unchanged DDSs) will need aggregate blob in previous summary
-                // that is no longer present in this summary.
-                // This is temporal limitation that can be lifted in future once BlobAggregationStorage becomes smarter.
-                const contextSummary = await context.summarize(true /* fullTree */, trackState);
+                const contextSummary = await context.summarize(fullTree, trackState);
```

Comment:
How do we know this works with `fullTree = false`? My understanding of the snapshot tests is that the tests don't test `fullTree = false`.

Reply:
There are some that do. 
packages\test\test-end-to-end-tests\src\test\summaries.spec.ts  . Ex. "Should generate summary tree" does it.